app/src/components/template/Dashboard.js

```diff
@@ -11,22 +11,37 @@ import {
 import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
 import MenuIcon from '@material-ui/icons/Menu';
 import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
 import { mainListItems } from './listItems';
 import { Styles as OBStyles } from '../../styles/OrderBookSnapshot';
 import OrderBookSnapshot from '../OrderBookSnapshot';
 import Home from '../Home';
 import NotFound from '../NotFound';
 import { styles } from '../../styles/Dashboard';
+import OrderBookService from '../../services/OrderBookService';
 
 class Dashboard extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
             open: false,
+            selectedInstrument: '',
+            instruments: [],
         };
     }
 
+    componentDidMount() {
+        const { instruments } = this.state;
+        const newInstruments = instruments.slice();
+        OrderBookService.getInstrumentsList().then(response => {
+            response.map(value => {
```

Comment:
In `axios` the response object from the server will hold the array at the following location in the response:
```
response.data;
```
Notice the `.data`. You would have to do `response.data.map(...)`

Reply:
thanks for pointing out